TACC/tup-ui
# feat(tup-cms): figure, caption, etc css; full site

## Overview

Apply figure and caption and blockquote styles to the whole site.

## Related

- requires https://github.com/TACC/Core-CMS/pull/624

## Changes

## Testing

1. Add figure and caption and blockquote and p.caption to a page (not a news article).
2. Verify the styles match those of [a news article](https://dev.tup.tacc.utexas.edu/news/latest-news/2025/11/11/basic-news-template/).

## UI

| sample test | use case test |
| - | - |
| ![figure, caption, etc](https://user-images.githubusercontent.com/62723358/230970186-bb73c03f-588c-421b-83d7-7220b2990546.png) | ![fake caption](https://user-images.githubusercontent.com/62723358/230970300-fdfc1c68-c1d1-4647-82aa-29e6c87ad968.png) |
